### PR TITLE
clarification on mTLS verification with curl

### PR DESCRIPTION
**Pull request by ldemailly**
this came from some confusion by a user on slack

### CODE REVIEW

File: _docs/tasks/security/mutual-tls.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ There are several steps:
    ```
    ```bash
    ...
+   * error fetching CN from cert:The requested data were not available.
```
Comment (ldemailly) [inline _docs/tasks/security/mutual-tls.md]: this to show the expected error

File: _docs/tasks/security/mutual-tls.md [hunk 2]
```diff
@@ -111,11 +113,15 @@ Note that Istio uses [Kubernetes service account](https://kubernetes.io/docs/tas
 as service identity, which offers stronger security than service name 
 (refer [here]({{home}}/docs/concepts/security/mutual-tls.html#identity) for more information). 
 Thus the certificates used in Istio do not have service name, which is the information that curl needs to verify
-server identity. As a result, we use curl option '-k' to prevent the curl client from verifying service identity
-in server's (i.e., productpage) certificate. 
+server identity. As a result, we use curl option '-k' to prevent the curl client from aborting when failing to
+find and verify the service identity
```
Comment (myidpt) [inline _docs/tasks/security/mutual-tls.md]: To avoid confusion, can you change "service identity" to "service name"?

Comment (ldemailly) [reply _docs/tasks/security/mutual-tls.md]: k maybe server name even ?

File: _docs/tasks/security/mutual-tls.md [hunk 3]
```diff
@@ -111,11 +113,14 @@ Note that Istio uses [Kubernetes service account](https://kubernetes.io/docs/tas
 as service identity, which offers stronger security than service name 
 (refer [here]({{home}}/docs/concepts/security/mutual-tls.html#identity) for more information). 
 Thus the certificates used in Istio do not have service name, which is the information that curl needs to verify
-server identity. As a result, we use curl option '-k' to prevent the curl client from verifying service identity
-in server's (i.e., productpage) certificate. 
+server identity. As a result, we use curl option '-k' to prevent the curl client from aborting when failing to
```
Comment (ldemailly) [inline _docs/tasks/security/mutual-tls.md]: @wattli I rephrased a bit - what do you think ?

Comment (wattli) [reply _docs/tasks/security/mutual-tls.md]: lgtm, thanks for fixing this up

**Pull request merged**